### ISSUE TITLE
Collision; upgrade `surface` dependency.

### ISSUE THREAD

**Issue by c4lliope** [surface-ui/surface_bulma#32]
```
Because surface_bulma >= 0.4.0 depends on surface ~> 0.7.0 and your app depends on surface ~> 0.11, surface_bulma >= 0.4.0 is forbidden.
So, because your app depends on surface_bulma ~> 0.4, version solving failed.
```

Sources:
* (0.4.0) https://hex.pm/packages/surface_bulma
* (0.11.0) https://hex.pm/packages/surface

**Comment by Zurga** [comment]
I will release a new version as soon as possible. Thanks for taking the time to report this.

**Comment by Zurga** [comment]
Should be fixed with the new release of 0.5.0

**Comment by c4lliope** [comment]
Looks good! I can add this as a dependency again.
There may be more issues to open as I explore the library, hope to help in any place I can.
